### ISSUE TITLE
Add option to "Add logo"

### ISSUE THREAD

**Issue by mariobehling** [fossasia/badgeyay#354]
1. In the beginning of the form add a section "Event Name", where the user can enter the name of the event as text.

2. After the section "Choose your Badge Background" add a section "Use Logo". Implement as follows:
* add radio buttons a) "Add event logo" b) Use event name text
* If user chooses "Add event logo" show an upload button for the image file

**Comment by S2606** [comment]
Should this event logo be places at middle of badge?? @mariobehling

**Comment by mariobehling** [comment]
Keep the same design as of now. White area on top. No changes.

**Comment by raju249** [comment]
I would like to work on this issue.
This will be my first issue with fossasia.
 
Can you assign me @mariobehling ?
Thanks

**Comment by djmgit** [comment]
@raju249 Nice to have you hear! Please go ahead :+1:

**Comment by raju249** [comment]
Thanks @djmgit 
Starting to work on this :smile:

**Comment by raju249** [comment]
This fix will also need backend implementation in flask for acknowledging changes.
Correct me if I am wrong.

**Comment by raju249** [comment]
@mariobehling  
The "Event Name" Section..will that be above "User Input Guide" or above "Choose your data Source" ?

**Comment by djmgit** [comment]
@raju249 you are right, you have to make necessary addition to the back end. Also you have to manipulate and update the svg base template to insert image name and logo.

**Comment by raju249** [comment]
Right @djmgit 
Got it.

**Comment by raju249** [comment]
@djmgit What about the sequence in which "Add Event Name" appears ?
Above or Below User Guide.
I think it should be below the user guide since we would want user to read the guide doing anything if he is not aware how the platform works.
Let me know what you think

**Comment by djmgit** [comment]
@raju249 Yeah, it should be below User Guide link, however once #367 is merged, user guide link will be replaced by a info icon on top right corner, so you need not worry about the position with respect to user guide. Just be sure to rebase your branch once #367 is merged.

**Comment by raju249** [comment]
Thanks @djmgit

**Comment by raju249** [comment]
@djmgit  
Can you help me of how do I put that Event Name or Logo on the badge's SVG base. 
Also what is the file name of the SVG that we should manipulate for this ?

Please bear with me :sweat_smile: I am learning on how badgeyay works.
Thanks for your time.

**Comment by djmgit** [comment]
@raju249 the file which you require to edit is /badges/8BadgesOnA3.svg. This is the base template. Open it in Inkscape and edit it.
When you create new elements, right click on the object to select object properties. Then provide suitable id and label sp that you can access the element from your python code.
Hope that helps.

**Comment by raju249** [comment]
Thanks @djmgit
Yes, we can edit using Inkspace.
But the event names will be different for every user.
How do we manipulate svg dynamically ?
Can you please elaborate what events are we talking about here ?

**Comment by raju249** [comment]
Ok let me guess.
When we edit the image using Inkspace we can give an id and label to part of the image where we will be inserting event name or logo. 
That is how we can change it dynamically right ?

**Comment by raju249** [comment]
@mariobehling 
Could you please elaborate on where to add the logo on the badge if user passes it ?
I am done with implementation of adding event name on badge.
Thanks

/cc @djmgit

**Comment by raju249** [comment]
Hey @harshit98,
Sure.
The thing is we have decided to keep this on hold since there is a discussion involved and there are other issues which have a higher priority which need to be solved. 😊

**Comment by ParthS007** [comment]
Working on this due to inactivity :+1

**Comment by raju249** [comment]
Thanks @ParthS007  for working on this.
Sorry to leave it out. Currently trying to fix some issues but also kinda busy with Internship 😄

**Comment by ParthS007** [comment]
will give a PR in 2 days 👍

**Comment by ParthS007** [comment]
will give a PR in 2 days 👍

**Comment by rupav** [comment]
@vaibhavsingh97 @ParthS007 @raju249 any of you working on 
- add radio buttons a) "Add event logo" b) Use event name text.
Would like to give it a try.

**Comment by ParthS007** [comment]
@rupav  I have worked on it as you know already but our first priority is to fix Travis, that's why I have kept PR on hold.

**Comment by rupav** [comment]
Ok @ParthS007 👍

**Comment by YogeshSharma0201** [comment]
@ParthS007 can i work on this issue?

**Comment by YogeshSharma0201** [comment]
@ParthS007 are you working on this?
